liquidator/liquidator.js

```diff
@@ -315,20 +318,15 @@ class Liquidator {
         liquidationResult: logResult
       });
     }
-
-    // Update the EMP Client since we created new liquidations.
-    await this.empClient.update();
   }
 
   // Queries ongoing liquidations and attempts to withdraw rewards from both expired and disputed liquidations.
-  async queryAndWithdrawRewards() {
+  async withdrawRewards() {
```

Comment:
I renamed this function as it no longer does any `update`. that's left for the `index.js`